pylint/checkers/stdlib.py

```diff
@@ -244,12 +244,6 @@
             "binascii.rlecode_hqx",
             "binascii.rledecode_hqx",
             "importlib.resources.contents",
```

Comment:
This should be moved to `(3, 11, 0)` as the deprecation was delayed.
https://docs.python.org/3/library/importlib.resources.html#importlib.resources.contents